acmed/config/nginx_hooks.toml

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2021 Rodolphe Bréard <rodolphe@breard.tf>
```

Comment:
Since you wrote this file, it should be your name on it, not mine.

Reply:
I'm not keen to this. It is your API that is just consumed. But anyway, will update accordingly.